.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,23 @@
-<!--- Hi, and thanks for contributing! -->
-<!--- Make sure to provide a general summary of your changes in the title above. -->
-<!--- For example: "[scsynth] Fix crash when encountering cute kittens" -->
+<!-- Please see CONTRIBUTING.md for guidelines. -->
 
-Purpose and Motivation
-----------------------
+## Purpose and Motivation
 
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to it here by writing "Fixes #555". -->
+<!-- If this fixes an open issue, link to it by writing "Fixes #555." -->
 
-Types of changes
-----------------
+## Types of changes
```

Comment:
when I submit PRs from the command line using hub, if the line starts with a # it will be removed, can you please leave them as-is?